### ISSUE TITLE
trian and test accuracy

### ISSUE THREAD

**Issue by yangly-simon** [Jacoo-Zhao/3D-Pose-Based-Feedback-For-Physical-Exercises#2]
I ran your code again. But there is a low train() and test accuracy.  Only about 30%, would you illustrate that?
<img width="1163" alt="image" src="https://github.com/Jacoo-Zhao/3D-Pose-Based-Feedback-For-Physical-Exercises/assets/135346133/c1abbf42-ad2d-4350-8955-3cea30575840">

**Comment by yangly-simon** [comment]
@Jacoo-Zhao

**Comment by Jacoo-Zhao** [comment]
@yangly-simon Hi, I am sorry for the late reply, how is it going now? It seems that your training process didn't work well as the speed was 1424.94 epoch/s with training loss super high. Could you please double-check and report?

**Comment by yangly-simon** [comment]
@Jacoo-Zhao I'm sorry that I still haven't solved it. As a newbie, I'm trying hard to understand the code logic. I ran the code in Colab and found that the train didn't work because the loss remained unchanged. (Looking at the screenshot in tensorboard)
Maybe the bug is caused by different package versions? I used the default package version in Colab.
Would you help me?
<img width="1339" alt="image" src="https://github.com/Jacoo-Zhao/3D-Pose-Based-Feedback-For-Physical-Exercises/assets/135346133/cd9c270f-0997-46ce-ac26-b15021fb4a46">

**Comment by JoonSeongLee** [comment]
@yangly-simon 
Hello, I am experiencing the same problem. Could you please tell me how to solve this?
